cmd/protecodeExecuteScan.go

```diff
@@ -336,5 +336,12 @@ func getTarName(config *protecodeExecuteScanOptions) string {
 	}
 	// replace unwanted chars
 	fileName = strings.ReplaceAll(fileName, "/", "_")
+	log.Entry().Infof("Filename : %s", fileName)
```

Comment:
Strange to have a log message `Filename: xxxx` in the log. Would be good to provide some context, e.g. about which kind of file are we talking ...

Reply:
removed as suggested